### PR TITLE
switch to govendor

### PR DESCRIPTION
**Pull request by termie**
ferengi version of #147

### CODE REVIEW

File: core/config.go [hunk 1]
```diff
@@ -78,6 +78,7 @@ func (d DockerAuth) ToAuthenticator(env *util.Environment) auth.Authenticator {
 }
 
 func (a AmazonAuth) ToAuthenticator(env *util.Environment) auth.Authenticator {
+	fmt.Printf("%v\n", env)
```
Comment (faiq) [inline core/config.go]: clean this

File: README.md [hunk 2]
```diff
@@ -30,15 +30,15 @@ up your go environment by
 Next, you'll need `glide` to install the golang dependencies. You can do this
 by running:
 ```
-  go get github.com/Masterminds/glide
+  go get github.com/kardianos/govendor
 ```
 
 In Go 1.5 you'll need the vendor experiment enabled, so make sure to export
 `GO15VENDOREXPERIMENT=1` in your shell (see [Go 1.5 Vendor Experiment](https://docs.google.com/document/d/1Bz5-UB7g2uPBdOx-rw5t9MxJwkfpx90cqG9AFL0JAYo/edit))
```
Comment (mwhooker) [inline README.md]: do we still need this? looks like it was removed in the wercker yaml

**Comment by mwhooker** [comment]
a few sentences on why we're switching to govendor would be helpful for context. other than that and faiq's and my comment above, lgtm

File: README.md (hunk 2 above)
Comment (termie) [inline README.md]: we're using 1.6+ in our builds but anybody on 1.5 still needs it

**Pull request merged**